src/main/resources/io/jenkins/plugins/orka/OrkaAgent/configure-entries.jelly

```diff
@@ -51,8 +51,13 @@
       <f:entry title="${%# of CPUs}" field="numCPUs">
         <c:select />
       </f:entry>
+
```

Comment:
We do not need this functionality for permanent agents as the user chooses the node here.
Feel free to revert the whole file and remove the help-scheduler.html file